libraries/AP_Proximity/AP_Proximity_LightWareSF40C.cpp

```diff
@@ -383,8 +383,8 @@ bool AP_Proximity_LightWareSF40C::process_reply()
         case RequestType_Health:
             // expect result in the form "0xhhhh"
             if (element_len[0] > 0) {
-                int result;
-                if (sscanf(element_buf[0], "%x", &result) > 0) {
+                long int result = strtol(element_buf[0], nullptr, 16);
```

Comment:
I'm not sure why this change to the proximity sensor is included in the AP_Filesystem PR.?   I haven't looked into the implications of the change from sscanf to strtol, and it's probably a harmless change, but thought I'd note it here anyway.